nexus-sdk-js/src/Project/index.ts

```diff
@@ -96,16 +100,29 @@ export default class Project {
   }
 
   async getResource(id: string): Promise<Resource> {
+    return await getResource(id, this.orgLabel, this.label);
+  }
+
+  // The current API does not support pagination / filtering of views
+  // This should be fixed when possible and converted to signituare
+  // Promise<PaginatedList<ElasticSearchView>>
+  async getElasticSearchViews(): Promise<ElasticSearchView[]> {
```

Comment:
I would rename this to `listElasticSearchViews`.

Reply:
you got it dude!

Reply:
also, can you add a `getElasticSearchView(id?: string = 'document')` method that returns the specific Elastic view (you need to manually check if it is an `ElasticView` in the types) or the default `ElasticView` if no ID is specified.

I will reflect that with `SparqlViews` methods.

Reply:
OK, looks like the api endpoint hasn't been implemented yet, so let's do this later. I've created an issue for it BlueBrain/nexus#333